cmd/deploy/deploy_test.go

```diff
@@ -96,17 +96,17 @@ func (*fakeKubeConfig) Read() (*rest.Config, error) {
 func (fc *fakeKubeConfig) Modify(_ int, _, _ string) error {
 	return fc.errOnModify
 }
-func (*fakeKubeConfig) GetModifiedCMDAPIConfig() (*clientcmdapi.Config, error) {
```

Comment:
Was this notation failing? I see it clearer than `func (_ *fakeKubeConfig) functionName`

Reply:
Nvm I just saw that you changed it in the next commit 😅